plugin/builtin/patchtransformer/PatchTransformer_test.go

```diff
@@ -198,6 +198,132 @@ Patch: "something"
 	})
 }
 
```

Comment:
Since `multiplepatch_test.go` tests the builr-in Kustomize fields `patches`, do you think we should add an additional test case in `PatchTransformer_test.go` that checks the `PatchTransformer` is able to patch multiple strategic merge patches successfully?

Reply:
I agree with your opinion!
I added those tests to https://github.com/kubernetes-sigs/kustomize/pull/5194/commits/b951c8e7c5f052813dcc81060da8aa7ffc62ee59.